test/src/helper/mock/tendermintMessage.ts

```diff
@@ -0,0 +1,257 @@
+// Copyright 2018-2019 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+import { H256, U64 } from "codechain-primitives";
+import { EventEmitter } from "events";
+import { compressSync, uncompressSync } from "snappy";
+
+const RLP = require("rlp");
+
+export const Emitter = new EventEmitter();
```

Comment:
Are you sure about a singleton `Emitter`?

Reply:
No, I'm not. But I have no better idea without refactoring the whole mock structure.